File: js/funnyString.js
let s = 'abc'
let sReverse = s.split('').reverse().join('')
let a1 = []
let a2 = []
for(let i=0; i<s.length; i++){
a1.push(s[i].charCodeAt(0))
a2.push(sReverse[i].charCodeAt(0))
}
let output = 'Funny'
for(let j=0; j<a1.length-1; j++){
    console.log(a2[j])
let difference = Math.abs(a1[j] - a1[j+1])
let difference2 = Math.abs(a2[j] - a2[j+1])
if(difference!=difference2){
    output = 'Not Funny'
    break;
}
}
console.log(output)

File: js/funnyString.cpp
#include<bits/stdc++.h>
using namespace std;
int main(){
string s= "acxz";
string reverse;
for(int j=s.length()-1; j> -1; j--){
reverse+= s[j];
}
for(int i=0; i<s.length()-1; i++){
if(abs(int(reverse[i])-int(reverse[i+1]))!=abs(s[i]-s[i+1])){
cout<<"Not Funny"<<"\n";
}
}
cout<<"Funny"<<"\n";
return 0;
}